runtime/common/src/claims.rs

```diff
@@ -147,18 +132,38 @@ impl sp_std::fmt::Debug for EcdsaSignature {
 	}
 }
 
-decl_event!(
-	pub enum Event<T> where
-		Balance = BalanceOf<T>,
-		AccountId = <T as frame_system::Config>::AccountId
-	{
+#[frame_support::pallet]
+pub mod pallet {
+	use frame_support::{pallet_prelude::*, traits::EnsureOrigin, weights::{Pays, DispatchClass}};
+	use frame_system::{pallet_prelude::*, ensure_signed, ensure_root, ensure_none};
```

Comment:
Importing just `pallet_prelude` should be enough:

```suggestion
	use frame_support::pallet_prelude::*;
	use frame_system::pallet_prelude::*;
```